torch/csrc/jit/tensorexpr/codegen.h

```diff
@@ -37,11 +41,20 @@ class CodeGen {
     return buffer_args_;
   }
 
+  at::Device device() {
+    return device_;
+  }
+
+  void set_device(at::Device device){
```

Comment:
One of my arbitrary quirks is that I think setters are generally to be avoided.  Why would a CodeGen not know its device at creation time?

Reply:
I'll get rid of this. I didn't notice this. It was left over from when I was trying to invoke this from the CPP unit tests.